README.md

```diff
@@ -94,5 +94,5 @@ make testacc
 To generate cassettes, run:
 
 ```shell
-TF_ACC=1 RENDER_OWNER_ID=<your owner id> RENDER_API_KEY=<your api key> UPDATE_RECORDINGS=true go test -count=1 -v
+TF_ACC=1 RENDER_OWNER_ID=<your owner id> RENDER_API_KEY=<your api key> RENDER_HOST=<render host> UPDATE_RECORDINGS=true go test -count=1 -v
```

Comment:
drive by fix?

Reply:
It's from commit https://github.com/render-oss/terraform-provider-render/pull/21/commits/7d4c534fcbb8f67643b47907775e674438a0424d with description "Document additional necessary environment variable for updating cassettes". Each commit in this PR is an independent change with its own description.